pkg/certificate/certificate_test.go

```diff
@@ -56,6 +58,9 @@ func getCertificateRequestForTest() *Request {
 
 	}
 	req.IPAddresses = ips
+
+	req.ExtKeyUsages = *NewExtKeyUsageSlice("ClientAuth")
```

Comment:
nit: may be a test with ServerAuth could be added, but not mandatory. just to have a different case.

Reply:
In fact that was a mistake... the ExtKeyUsage in the slice must have been "ServerAuth"

Reply:
Fixed now